Gruntfile.js

```diff
@@ -31,11 +32,14 @@ module.exports = (grunt) => {
   function webpackConfig(target, preset, mode) {
     return {
       mode,
-      entry: MODULE_ENTRY,
+      entry: {
+        vexflow: VEXFLOW_MODULE_ENTRY,
+        webComponents: WEB_COMPONENTS_MODULE_ENTRY,
+      },
       output: {
         path: BUILD_DIR,
-        filename: target,
-        library: 'Vex',
+        filename: (mode === 'production') ? '[name]-min.js' : '[name]-debug.js',
```

Comment:
How is this change related to the web components additions? Is it just a quality-of-life improvement for development?

Reply:
Because there's two entry points now (one for the vexflow index.js, one for the web components index.js), the `[name]` syntax will create an output file for each, as opposed to just to `target`. I'm not sure what would happen if I left it just as one output file... but I do specifically import from the generated `webComponents-debug` file in one of the scripts that generates test images.

Reply:
ah, got it